packages/terra-status-view/src/terra-dev-site/doc/status-view/DeprecationGuide.2.doc.mdx

```diff
@@ -0,0 +1,12 @@
+## Deprecation Notice
+The terra-status-view npm package is deprecated as of `4.41.0`. terra-status-view will be removed from the repo in a future version, but the package will remain on npm.
+
+### What deprecation means
+Any apps that currently use terra-status-view will continue to work as-is.
+However, the terra-status-view package will no longer continue to be supported / developed.
+
+## Why?
+The reason behind terra-status-view deprecation is to integrate the presentation of overlays with the overall application framework to ensure better accessibility.
+
+## How to replace this package?
```

Comment:
we should probably mention that the ApplicationStatusOverlay is constrained to baked in variants to promote a unified framework pattern, so custom variants will not be supported moving forward

Reply:
Updated here https://github.com/cerner/terra-core/pull/3363/commits/5acc29b010ed85464bfaabcbed46c82802ee2cb4